modules/loader/include/loader/elf.h

```diff
@@ -266,6 +271,7 @@ class ElfFile {
     addr_t value;
     addr_t size;
   };
+  static_assert(sizeof(Symbol64) == 0x18, "Wrong Symbol size");
```

Comment:
Not necessary to make it part of this PR, but would it make sense to have a `Symbol<addr_t>` template class?

Reply:
Oh wait, hadn't clocked that the fields are in a different order, it wouldn't work, never mind.